config/initializers/rollbar.rb

```diff
@@ -0,0 +1,53 @@
+Rollbar.configure do |config|
+  config.access_token = Rails.application.secrets.rollbar_token
+
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+  config.enabled = Rails.env.production? && config.access_token.present?
+
+  # By default, Rollbar will try to call the `current_user` controller method
+  # to fetch the logged-in user object, and then call that object's `id`,
+  # `username`, and `email` methods to fetch those properties. To customize:
+  config.person_method = nil
```

Comment:
One thought I had over lunch hour: since our view into the production environment is limited, we may not be able to map the id field to individual user records (at least not directly, without consulting a sys admin first).